equinity-historical-data/src/main/java/uk/gov/justice/laa/crime/equinity/historicaldata/model/CrmFormModelInterface.java

```diff
@@ -0,0 +1,6 @@
+package uk.gov.justice.laa.crime.equinity.historicaldata.model;
+
+public interface CrmFormModelInterface {
```

Comment:
Do we specifically needs to mention Interface? - CrmFormModelInterface

Reply:
I find it easier to identify which are interfaces and which aren't when reading the code.